python/ray/_private/serialization.py

```diff
@@ -493,10 +528,38 @@ def _python_serializer(o):
         msgpack_data = MessagePackSerializer.dumps(value, _python_serializer)
 
         if python_objects:
-            metadata = ray_constants.OBJECT_METADATA_TYPE_PYTHON
-            pickle5_serialized_object = self._serialize_to_pickle5(
-                metadata, python_objects
-            )
+            contains_adag = False
```

Comment:
This seems like it was added by mistake?

Reply:
Yep, thanks.